README.md

```diff
@@ -283,7 +283,7 @@ await producer.produce({
 const headers = memphis.headers();
 headers.add('<key>', '<value>');
 await producer.produce({
-    message: '<bytes array>', // Uint8Arrays / You can send object in case your station is schema validated
+    message: '<bytes array>/object/string/DocumentNode graphql', // Uint8Arrays / You can send object/string/DocumentNode graphql in case your station is schema validated
```

Comment:
fix the comment so that it will show which types can be sent to which schema type, as you wrote in python